requirements.txt

```diff
@@ -1,2 +1,4 @@
 brainatlas-api
-napari[pyqt5]
\ No newline at end of file
+napari[pyqt5]
+pyqt5
```

Comment:
If we have the `napari[pyqt5]` dependency, it will also install a napari-compatible version of pyqt5, which I think would be better than requiring it ourselves (in case we get the wrong version).

Reply:
perfect will remove it thanks